packages/sdk/src/Confidence.ts

```diff
@@ -56,6 +56,8 @@ export interface Configuration {
   /** Flag Resolver Client
    * @internal */
   readonly flagResolverClient: FlagResolverClient;
+  /* @internal */
+  readonly clientSecret: string;
```

Comment:
is it possible to declare it `private`? (not sure it helps but still)

Reply:
I don't think it's possible on an interface